ts/cowfee.ts

```diff
@@ -164,15 +162,19 @@ export const swapTokens = async (
     throw new Error(`appData mismatch: ${appDataHex} != ${config.appData}`);
   }
 
-  const toSwapWithBuyAmount = toSwap.map((token) => {
-    const buyAmount = token.tokenOut
-      .mul(10000 - config.buyAmountSlippageBps)
-      .div(10000);
-    return {
-      ...token,
-      buyAmount: buyAmount.eq(0) ? BigNumber.from(1) : buyAmount,
-    };
-  });
+  const toSwapWithBuyAmount = toSwap
+    .map((token) => {
+      const buyAmount = token.tokenOut
+        .mul(10000 - config.buyAmountSlippageBps)
+        .div(10000);
+      return {
+        ...token,
+        buyAmount,
+      };
+    })
+    .filter((token) => {
+      return token.buyAmount.gt(config.minOut);
+    });
```

Comment:
I think it would be nice if the script could log what tokens are originally considered and for what reason certain tokens are filtered out (to get better observability when it runs but doesn't produce the amount of orders we expect).